test/test_models.py

```diff
@@ -893,7 +895,7 @@ def test_quantized_classification_model(model_fn):
     out = model(x)
 
     if model_name not in quantized_flaky_models:
-        _assert_expected(out, model_name + "_quantized", prec=2e-2)
```

Comment:
Do we still need the `.cpu()` call here, if this is done anyway in the comparison logic by `check_device=False`? This applies to all occurrences of this.

Reply:
If we don't add it, it will take something estimated in cuda and produce an expected file. This might cause issues to users who load it without cuda and might require to map it to CPU. That's why I put it everywhere it should be. Might be an overkill.